### PR TITLE
Filters and product listing page

### PR DESCRIPTION
**Pull request by vanshitalavange**
- Created product listing page with filters
screencapture-127-0-0-1-5501-products-html-2022-02-19-02_09_39
- Live preview: https://deploy-preview-3--the-organic-store.netlify.app/products.html

### CODE REVIEW

File: stylesheets/pages/products.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  grid-template-columns: 1fr 4fr;
+  gap: 0 var(--spacing-sm);
+  width: 90vw;
+  margin: var(--spacing-md) auto;
+}
+/** filters **/
+.filters {
+  gap: var(--spacing-xs);
+  box-shadow: 1px 2px 10px rgb(238, 234, 234);
+  min-width: 17rem;
```
Comment (Gautam2305) [inline stylesheets/pages/products.css]: you usually don't want to change the width of filters sidebar when screen size changes......so giving a max -width might be more relevant here in this case than giving a min-width or just add a max-width to this code too accordingly

**Review by Gautam2305 — COMMENTED** [review]
Rest everything looks clean. Good use of minmax property & variables for spacing etc.

File: stylesheets/pages/products.css (hunk 1 above)
Comment (vanshitalavange) [reply stylesheets/pages/products.css]: Yes, that makes sense to me. Changes done, please approve

**Comment by sakleprajwal** [comment]
In live preview, when I click on radio and checkbox input, it works fine. But when I click on the respective label, it does not get selected. You need to use for attribute in the label.

**Review by sakleprajwal — APPROVED** [review]
I have suggested a change in the comment. Everything else looks good. Please go ahead.

**Comment by vanshitalavange** [comment]
> In live preview, when I click on radio and checkbox input, it works fine. But when I click on the respective label, it does not get selected. You need to use for attribute in the label.
Thank you for suggesting this! I wasn't knowing about it. Changes done

**Pull request merged**